tests/client-dbus/tests/udev/test_udev.py

```diff
@@ -58,19 +60,33 @@ def random_string(length):
     )
 
 
-def _create_pool(name, devices):
+def _create_pool(name, devices, *, key_description=None):
```

Comment:
Yes. I'ld like to force keyword only args where they really are keywords. There's a subtle bug in pyudev somewhere, because this is not done: https://github.com/pyudev/pyudev/issues/243. Also, the Hypothesis people enforce this everywhere, and they pay way more attention to the state of Python than I do: https://github.com/HypothesisWorks/hypothesis/pull/2317.

Reply:
Okay that makes sense.